hw/ip/rom_ctrl/doc/_index.md

```diff
@@ -114,7 +114,8 @@ Since the code in ROM is the first thing to execute, an attacker that modifies i
 As such, OpenTitan needs some form of ROM integrity checking and the ROM checker is the module in charge of providing it.
 
 After bringing the ROM controller module out of reset, the power manager must wait until `pwrgr_data_o.done` is asserted before starting the host processor.
-In PROD state, it will also check that `pwrmgr_data_o.good` is `On`: if not, it will refuse to boot.
+The ROM controller also passes the `pwrmgr_data_o.good` signal.
```

Comment:
FYI: This intersects with https://github.com/lowRISC/opentitan/pull/9773

Reply:
Thanks! I'll have a look there in a minute.